app/core/src/test/java/com/fsck/k9/notification/NewMailNotificationManagerTest.kt

```diff
@@ -173,7 +183,8 @@ class NewMailNotificationManagerTest {
         }
 
         assertNotNull(result) { data ->
-            assertThat(data.cancelNotificationIds).containsExactly(
+            assertThat(data.cancelNotificationIds).containsExactlyInAnyOrder(
+                // FIXME is the order important? If yes, use containsExactly
```

Comment:
I had to change this test to `containsExactlyInAnyOrder` as the order doesn't match the expected one. Is the order important?